src/Common/JemallocNodumpAllocatorImpl.cpp

```diff
@@ -0,0 +1,84 @@
+#include <Common/JemallocNodumpAllocatorImpl.h>
+
+#if USE_JEMALLOC
+#include <fmt/format.h>
+#include <base/errnoToString.h>
+#include <base/defines.h>
+#include <sys/mman.h>
+#include <cstring>
+
+namespace DB
+{
+
+JemallocNodumpAllocatorImpl & JemallocNodumpAllocatorImpl::instance()
+{
+    static JemallocNodumpAllocatorImpl allocator;
+    return allocator;
+}
+
+void * JemallocNodumpAllocatorImpl::allocate(size_t size, size_t alignment) const
```

Comment:
Am I missing something or memory from `JemallocNodumpAllocatorImpl` is not tracked by `MemoryTracker`? (`CurrentMemoryTracker::alloc`/`CurrentMemoryTracker::free`)